simulator/carla_autoware/README.md

```diff
@@ -0,0 +1,58 @@
+# CARLA_Autoware
+
+# ROS2/Autoware.universe bridge for CARLA simulator
+
+Thanks to <https://github.com/gezp> for ROS2 Humble support for CARLA Communication.
+This ros package enables communication between Autoware and CARLA for autonomous driving simulation.
+
+# Environment
+
+| ubuntu |  ros   | carla  |    autoware     |
+| :----: | :----: | :----: | :-------------: |
+| 22.04  | humble | 0.9.15 | universe/master |
+
+# Setup
+
+## install
+
+- [Autoware.Universe](https://autowarefoundation.github.io/autoware-documentation/galactic/installation/autoware/source-installation/)
+- [CARLA Installation](https://carla.readthedocs.io/en/latest/start_quickstart/)
+- [Carla Lanelet2 Maps](https://bitbucket.org/carla-simulator/autoware-contents/src/master/maps/)
+- [Autoware launch with CARLA option](https://github.com/mraditya01/autoware_launch)
```

Comment:
You should remove this line. `autoware_launch` does not need any special setup (once the PR is merged).